app/shop.py

```diff
@@ -0,0 +1,39 @@
+import datetime
+
+from app.customer import Customer
+
+
+class Shop:
+    def __init__(
+            self,
+            name: str,
+            location: list[int],
+            products: dict[str, int]
+    ) -> None:
+        self.name = name
+        self.location = location
+        self.products = products
+
+    def print_check(self, customer: Customer) -> None:
+        current_time = datetime.datetime.now()
+        current_time = current_time.strftime("%d/%m/%Y %H:%M:%S")
```

Comment:
you can do it in one line

`datetime.datetime.now().strftime("%d/%m/%Y %H:%M:%S")`